src/patterns/news/index.njk

```diff
@@ -0,0 +1,87 @@
+---
+title: News
+group: Pages
+---
+{% from "views/partials/example/_macro.njk" import patternlibExample %}
+{% from "components/external-link/_macro.njk" import onsExternalLink %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+How to display, tag and categorise news articles.
+
+{{
+    patternlibExample({ "path": "patterns/news/examples/landing/index.njk" })
+}}
+
+## When to use this pattern
+
+Use this pattern to publish a news article to a public-facing website.
+
+## How to use this pattern
+
+Use the [articles component](/components/articles) to build consistent articles.
+
+### News index
+
+This index sorts the already-published news so that the most recently posted article is at the top of the list.
+
+Each article listed on the news index needs to have a:
+
+- short, clearly written title that links to the [news article](/patterns/news/#news-article)
+- short summary, usually taken from the first paragraph of the article, which should be no more than four lines (around 150 characters)
+- [publication date that follows house style](https://style.ons.gov.uk/house-style/dates), for example, 23 September 2020
+- thumbnail image
+
+A list of categories appears on the right-hand side of the news index, and you could also add contact details for the media team. Another optional feature is “[Follow us](https://ons-design-system.netlify.app/styles/typography/#list-with-social-icons)” social media links to Instagram, Facebook, Twitter and LinkedIn.
```

Comment:
Don't think we should say "right-hand side" because that's not the case on all screen widths.  

Worth linking the word 'categories' to the section below.